srcJava/com/rti/perftest/ddsimpl/PerftestTransport.java

```diff
@@ -992,6 +992,9 @@ public boolean configureTransport(DomainParticipantQos qos) {
     public String getMulticastAddr(String topicName)
     {
         //get() function return null if the map contains no mapping for the key
+        if (multicastAddrMap.get(topicName) == null) {
+            return null;
+        }
         return multicastAddrMap.get(topicName).toString();
```

Comment:
When the topic name does not match, the map return a null. then null.toString() will fail.